### PR TITLE
Fix saving of meeting and Issues

### PR DESCRIPTION
**Pull request by lbhssandilya**
Changed model, corrected Ids,changed testcases

### CODE REVIEW

File: ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs [hunk 1]
```diff
@@ -125,8 +125,8 @@ public async Task<CreateETRAMeetingActionResponse> CreateETRAMeeting(ETRAIssue m
             {
                 JObject tmiJObject = await CreateTenancyManagementInteractionJObject(meetingInfo, incidentid, ticketnumber);
 
-                try
-                {
+                //try
+                //{
```
Comment (bjpirt) [inline ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs]: Is this intentional?

Comment (bjpirt) [reply ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs]: Actually, I think it was me that commented this out :-)

Comment (bjpirt) [reply ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs]: The cause of the exception was being swallowed - we should probably revert this commit

Comment (lbhssandilya) [reply ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs]: I think it was in your code I haven't done any changes in exception catching/logging

Comment (bjpirt) [reply ManageATenancyAPI/Actions/Housing/NHO/ETRAMeetingsAction.cs]: reverted

**Pull request merged**